cmake/tests.cmake

```diff
@@ -4,6 +4,7 @@ endif()
 
 option(protobuf_ABSOLUTE_TEST_PLUGIN_PATH
   "Using absolute test_plugin path in tests" ON)
+mark_as_advanced(protobuf_ABSOLUTE_TEST_PLUGIN_PATH)
```

Comment:
Why not? It is only used to build and run tests from the cmake project. This path is not used for the native build process (Linux).
I believe that this is an advanced setting.